api/src/pcapi/utils/rate_limiting.py

```diff
@@ -36,6 +37,18 @@ def get_username_or_remote_address() -> str:
     return get_remote_address()
 
 
+class Limiter(flask_limiter.Limiter):
+    """A custom class that can completely disable rate-limiting if a
+    feature flag is enabled.
+    """
+
+    # pylint: disable=unused-private-member
+    def __check_request_limit(self, in_middleware: bool = True) -> None:
+        if not settings.IS_RUNNING_TESTS and not FeatureToggle.WIP_ENABLE_RATE_LIMITING.is_active():
+            return None
+        return super().__check_request_limit(in_middleware=in_middleware)
+
+
```

Comment:
Une suggestion, `__init__` de Limiter prend un booléen `enabled`, autant s’en servir non ? Sinon je veux bien comprendre

```suggestion
rate_limiter = Limiter(
    strategy="fixed-window-elastic-expiry",
    key_func=get_remote_address,  # The default is a deprecated function that raises warning logs
    enabled=not settings.IS_RUNNING_TESTS and not FeatureToggle.WIP_ENABLE_RATE_LIMITING.is_active()
)
```